### ISSUE TITLE
Request: Initial SQL Read/Write Support

### ISSUE THREAD

**Issue by r-richmond** [tableau/document-api-python#110]
Certain Datasources allow for initial sql to be passed. Being able to programmatically modify it would be extremely useful.

**Comment by t8y8** [comment]
@r-richmond Thank you for opening a request!

Would it be possible to include a dummy-tds or twb that already has some initial SQL in it? 
Additionally, an example update you may want to make is also great!
It's very helpful for us when building test cases and understand the requirements.

(I'm marking this as an enhancement)

**Comment by r-richmond** [comment]
Sure,

I've attached a sample workbook in #109 that has both query bands and initial sql.

As an example use case I'd like to be able to write something like following;

```
for workbook in folder:
  for datasource in workbook:
    if datasource.initial_sql is none:
      datasource.initial_sql = 'insert into schema.test_table values(current_timestamp,' + workbook.name + ');'
   elif datasource.initial_sql does not contain 'schema.test_table':
      datasource.initial_sql += '\n 'insert into schema.test_table values(current_timestamp,' + workbook.name + ');'
```

**Comment by r-richmond** [comment]
Marking this as closed with release 0.6 & the merging of #123
